topology/Internet2/Internet2Chicago/I2ChicagoInfrastructure.yaml

```diff
@@ -0,0 +1,26 @@
+Production: true
+SupportCenter: Community Support Center
+GroupDescription: Services located at the Internet2 Chicago POP
```

Comment:
What's a "POP"?

Reply:
"Point of Presence"

Reply:
Could you spell that out in all of the descriptions?